docs/en/secops/installation.asciidoc

```diff
@@ -0,0 +1,42 @@
+[[install-sec-monitoring]]
+[role="xpack"]
+== Get up and running
+
+beta[]
+
+To get up and running with security monitoring, you need:
+
+* An Elasticsearch cluster and Kibana (version 6.x or later) with a basic
+license. To learn how to get started quickly, see
+{stack-gs}/get-started-elastic-stack.html[Getting started with the {stack}].
++
+[TIP]
+==============
+You can skip installing {es} and {kib} by using our
+https://www.elastic.co/cloud/elasticsearch-service[hosted {es} Service] on
+Elastic Cloud. The {es} Service is available on both AWS and GCP.
+https://www.elastic.co/cloud/elasticsearch-service/signup[Try the {es}
+Service for free].
+==============
+
+* {beats} shippers (version 6.x or later) installed on each system you want to
+monitor
+
+You might need to modify UI settings in {kib} to change default behaviors,
+such as the index pattern used to query the data, and the timestamp field used
+for sorting. For more information, see {kib}.
+
+[float]
+[[install-beats-for-sec]]
+=== Install {beats} shippers
+
+To populate the security UI with metrics and
+log data, you need to install and configure the following shippers:
+
+* https://www.elastic.co/products/beats/packetbeat[{packetbeat}] for analyzing
+network packets 
+* https://www.elastic.co/products/beats/filebeat[{filebeat}] for forwarding and
+centralizing logs and files
+* https://www.elastic.co/products/beats/auditbeat[{auditbeat}] for monitoring
+directories for file changes
+
```

Comment:
I'd like to split this apart and describe what the options are for host security monitoring and the options for the network security monitoring. We could list the products and what data they provide.

Host Security Monitoring

- Auditbeat
- Filebeat
  - [System Module](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-system.html) - Authentication logs.
  - [Santa Module](https://www.elastic.co/guide/en/beats/filebeat/master/filebeat-module-santa.html) - Process execution monitoring for MacOS.

Network Security Monitoring
- Filebeat Bro Module
- Filebeat Suricata Module
- Filebeat Netflow Input
- Filebeat IPTables Module
- Packetbeat
- Auditbeat System Socket Module 
- Any ECS compatible data source

Another way to present the information would be to turn it around and list the things you can monitor like processes, packages, and flows; and then list all things that can be used to monitor them. This might present better as a matrix with the features on the left and products across the top.

Host Security Monitoring

- Processes - Auditbeat / Filebeat Santa
- Network Connections - Auditbeat System Sockets Module / Winlogbeat + Sysmon
- System Changes - Auditbeat System Host Module
- Users - Auditbeat System User Module
- Logins / Logoffs - Auditbeat Login Module
- Packages - Auditbeat System Package Module

Network Security Monitoring

- Flows - Packetbeat / Filebeat Netflow / Filebeat Bro / Filebeat Suricata
- Firewalls
  - IPTables - Filebeat
- Deep Packet Inspection
  - DHCP - Packetbeat
  - DNS - Packetbeat / Filebeat Bro / Filebeat Suricata
  - HTTP - Packetbeat / Filebeat Bro
  - TLS - Packetbeat / Filebeat Bro
  - ICMP - Packetbeat

@karenzone I'll let you decide on what's the best way to present the information. I can help with filling in the details. We could merge this as and then continue to iterate from here or make the changes now.